sdkgen/nodejs-request/lib/index.js

```diff
@@ -1,21 +1,23 @@
 // TODO add options and fetch options
-const getCodegenOptions = require('postman-code-generators').getOptions,
-  processCollection = require('./util').processCollection,
-  sanitize = require('./util').sanitize;
+const processCollection = require('./util').processCollection,
+  sanitize = require('./util').sanitize,
+  combineVariablesLists = require('../../../lib/utils').combineVariablesLists;
 
 /**
  * Generates sdk for nodejs-request
 
  * @param {PostmanCollection} collection - Postman collection Instance
+ * @param {PostmamVariableList} variables - Postman Variable List with resolved scope
  * @param {Object} options - postman-code-generators options
  * @param {Function} callback - callback functio to return results (err, response)
  * @returns {String} - sdk snippet for input collection
  * TODO add indentation to entire snippet
  */
-function generate (collection, options, callback) {
+function generate (collection, variables, options, callback) {
   var snippet = '',
     indent = options.indentType === 'Tab' ? '\t' : ' ';
   indent = indent.repeat(options.indentCount);
+
   processCollection(collection, options, (err, collectionSnippet) => {
```

Comment:
processCollection doesn't have a callback right?

Reply:
In this version it has. This thing has been revised in #3